project.mml

```diff
@@ -1457,6 +1458,7 @@ Layer:
             OR historic IN ('memorial', 'monument', 'archaeological_site')
             OR tags->'memorial' IN ('plaque')
             OR highway IN ('bus_stop', 'elevator', 'traffic_signals')
+            OR tags->'parking' IN ('underground')
```

Comment:
I don't see that we need to change the WHERE clause

Reply:
What do you suggest tu put here?

Reply:
Nothing.

Reply:
Indeed, this case should be already covered by "OR amenity IS NOT NULL"